pwa-devdocs/src/pagebuilder/known-limitations/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Known limitations
+---
+
+The following items are known limitations to implementing PWA components for Page Builder content types:
+
+- **Products content type** - The Alignment property as set within the Products content type form will always default to Left. Text alignment does not work within the PWA Studio gallery because the gallery is rendered with the CSS grid layout.
+
+- **Text content type** - Widgets are not supported within Text components in PWA Studio.
```

Comment:
This isn't just the Text content type, this is for any content type that supports text input (Text, Slider, Banner)